charts/pega/templates/pega-traefik-config.yaml

```diff
@@ -0,0 +1,8 @@
+{{ $depName := printf "%s" (include "deploymentName" $) }}
+
+{{ if (eq (include "performDeployment" $) "true") }}
+{{ range $index, $dep := .Values.global.tier }}
+{{ $data := dict "root" $ "node" $dep "depname" $depName  "name" (printf "%s-%s" $depName $dep.name) }}
+{{- include "pegaTraefikConfigTemplate" $data }}
```

Comment:
Is it required to add below conditions here as well?
```
{{- if .node.tls }}
{{- if .node.tls.enabled }}
{{- if .node.tls.traefik.enabled }}
```

Reply:
The condition is present inside the template